resume.md

```diff
@@ -1,49 +1,52 @@
-# FIRST LAST NAME
+# Adithya Kumar
 
-(123) 456 - 7890 | email@gmail.com
+(512) 660 - 1911 | adithya.shak.kumar@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/shakstzy | https://www.linkedin.com/in/adithyashakthikumar/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+University of California, Berkeley | B.S. Electrical Engineering & Computer Science + Business ... *May 2023*
```

Comment:
ommit `+` is it something like Business and Administration or somethign? idk no idea